README.md

```diff
@@ -2,24 +2,47 @@
 
 # Databraid Slack API
 
-## Usage
 
-Before beginning make sure you have [Docker](https://www.docker.com/) installed and running, then do:
+## Description
 
-```
-npm i
-npm run up
-```
+An Express server/router which serves as the backend for the Databraid Slack Widget.  This is used in conjunction with the Databraid Slack SPA.  It is built to run as a container within Docker.
 
-When finished developing do:
+The Databraid Slack API handles all interaction with Slack's Event API.  It authenticates with Slack during an OAuth sign-in.  It makes requests to the Slack Event API to fetch information about a Slack team's users and channels.  It interfaces with a PostgreSQL database to store and retrieve data. It interfaces with Google's Natural Language Processor API to perform sentiment analysis on the channel's content. It is also responsible for setting up a web socket and, once the Databraid Slack SPA connects with that socket, pushing new event and sentiment information to the client.
+
+## Usage
+
+There are a number of things that need to be in place prior to having this server function correctly.  You must:
+
+- Have a __Slack team__ to which you hold administrative privileges.
+- Set up a __Slack App__ at [api.slack.com](https://api.slack.com/) which listens for new events for that team and pushes these to the client.  You will also need to verify the server with the Slack App using the servers `/slack/events` endpoint.
+- Get a __Google Account__ and license key for Google NLP API.
```

Comment:
@Dave-Gallup if you can provide a link here that will be very helpful

Reply:
I think I did this, now.